source/pastemyst/util/username.d

```diff
@@ -2,112 +2,153 @@ module pastemyst.util.username;
 
 import std.string;
 
-private const(string) alpha = "abcdefghijklmnopqrstuvwxyz";
-private const(string) nums = "0123456789";
-private const(string) alphanums = alpha ~ nums;
-private const(string) symbols = "-_.";
```

Comment:
Just an hint here @CodeMyst . The compiler converts this value type to immutable values. If you do const(T) the compiler will do immutable(const (T)). You should use immutable or enum for compile-time constant values